### ISSUE TITLE
fix(model): publish quantized hybrid Step target hidden state

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#12859]
Parent: #12349
Coordinates with: #12238
Coordinates with: #12717

## Parent context
#12349 requires exact serial logits and state for native Metal MTP verification.

## Current state
The strict physical witness retains 36 target hidden rows after P4 Verify and Commit but reports a serial history mismatch. Read-only inspection finds the quantized hybrid tokenHiddenQ branch appends cache position and returns finalNorm without rememberTargetHidden. Runtime serial counts were not captured; prefix32 retention is a source inference.

## For / Problem / Today / Better because
For native quantized Qwen MTP callers, publish raw hidden state for each completed serial Step. Today the F32 serial path publishes while the quantized hybrid path omits publication. Reuse the existing position-bound publication helper so callers can retrieve the committed residual stream.

## Why this is next
Repair observable serial bookkeeping before interpreting three-way old-serial, graph-P1 and P4 exact state comparisons.

## Working spine
CPU synthetic quantized hybrid session -> captured prefix -> Step -> committed position and token history -> exact raw hidden retrieval, with a reproduction that fails against unchanged production.

## Core through-line
One missing raw-hidden publication at the existing quantized hybrid completion seam; no arithmetic change.

## Scope
```routing
lane: model
paths: ["internal/model/quant_forward.go", "internal/model/quant_target_hidden_test.go"]
expected_steps: 3
```
Separate isolated worktree. Bind the frozen #12717 quant_forward.go dependency explicitly; never edit that live candidate. Its graph route declines captureTargetHidden and remains unchanged.

## Scope class
Work unit: leaf
Single S1 bookkeeping regression and minimal publication fix.

## Definition of done
- [ ] Independently reviewed two-file source packet and dependency hashes.
- [ ] Same focused regression fails before and passes after in granted CPU window.
- [ ] Independently witnessed integration and explicit-path commit.

## Gold-plating boundary
No shaders, normalization changes, tolerance, graph capture extension, wider admission, or claim that this fixes numerical parity. Original strict RED remains recorded.

## Acceptance gate
Reproduction-first exact token/position/raw-hidden assertions; source review before execution. No CPU or HIL without campaign grant.

## Done condition
Quantized hybrid Step publishes the committed raw hidden row through the existing helper, verified by one focused regression; parent remains open for exact numerical qualification.

## Witness
CGO_ENABLED=0 GOMAXPROCS=2 go test -p=1 -timeout=60s ./internal/model -run '^TestQuantizedHybridStepPublishesTargetHidden$' -count=1

## Duplicate search
Open and closed searches for hidden capture quantized, targetHidden Step, and rememberTargetHidden found no matching serial publication defect. #12720 concerns Vulkan raw target history, #12853 isolates normalization.

## Problem frame
Centrality: Core
Portfolio tier: 2 serving
P1: advanced - publish position-bound committed hidden state.
P2: preserved - no performance claim.
P3: preserved - two files and existing helper.
P4: advanced - reproduction binds observable state.

## Work estimate
Estimate: 1 point.

## Overall completion contribution
Contribution: 1/4 points within #12349; planning estimate only.

## Completion standard
development

## Closure binding
Explicit-path verified commit cites this child and (fak model); no parent closure until exact original acceptance passes.

**Comment by anthony-chaudhary** [comment]
Captured CPU behavioral RED/GREEN on frozen #12717 v4 plus #12843 dependencies. The unchanged TestQuantizedHybridStepPublishesTargetHidden first failed because committed position 2 had no raw hidden row (session 57196, exit 1, 0.457s). One rememberTargetHidden call before finalNorm then passed the same test (session 70978, exit 0, 0.436s). Authored two-file patch SHA256 6787879dd7793eb9361478a1f7eba36e8868daf98cde55cd465216031bdc1414. Test hash unchanged 8d24711d73be2ab7cf7de2c6655f5125c75bf7edefe423682ce7b35309c3fc1e; full source manifests differ only in quant_forward.go. Final source review and integration pending; no commit or closure claim. This proves CPU synthetic Q8 serial hidden publication only. Original #12349 strict Metal numerical parity remains RED. CPU slot and canonical lease released.
